platform/mds/mds/src/main/java/org/motechproject/mds/service/impl/EntityServiceImpl.java

```diff
@@ -882,6 +883,40 @@ public String paramTypeForQuery() {
 
     @Override
     @Transactional
+    public List<EntityDto> findEntitiesBySuperclass(String classnameSuper) {
+        List<EntityDto> entities = new ArrayList<>();
+
+        FilterValue filterValue1 = new FilterValue() {
```

Comment:
Filters are generally only used for the filter part of the MDS data browser. There are better ways to do this, like simply using a query or one of the retrieve methods.